comicrelief/campaign
# [#1309] - Add news editor role

Fixes #1309

## How to test
* Login `news_editor` : `news_editor`
* Users with new "News editor" role can create and edit news articles including access to entities required for those (e.g. media, pdf..). 
* They should see the administration theme when editing the content
* They shouldn't be able to do anything else.
